2-extra/5-journey-planner.js

```diff
@@ -19,7 +19,20 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
+
+function journeyPlanner(locations, transportMode) {
+  let whereToGo = []
+  
+  for (const [key, value] of Object.entries(locations)){
```

Comment:
what is key and value in this case? Remember to name the variables so it's easier to identify the purpose of them.